src/Console/CommandRunner.php

```diff
@@ -50,6 +58,26 @@ public function __construct(BaseApplication $app, $root = 'cake')
     {
         $this->app = $app;
         $this->root = $root;
+        $this->aliases = [
+            '--version' => 'version'
+        ];
+    }
+
+    /**
+     * Replace the alias map for a runner.
+     *
+     * Aliases allow you to define alternate names for commands
+     * in the collection. This can be useful to add top level switches
+     * like `--version` or `-h`
+     *
+     * @param array $aliases The map of aliases to replace.
+     * @return $this
+     */
+    public function setAliases(array $aliases)
+    {
+        $this->aliases = $aliases;
```

Comment:
I don't see why we would have to change it. As @ravage84 mentioned we could add methods like `addAliases()` and `removeAlias()` if we need to add/remove into the existing set of aliases.